src/main/resources/view/MainWindow.fxml

```diff
@@ -16,7 +16,7 @@
 
 <fx:root minHeight="600" minWidth="450" onCloseRequest="#handleExit" title="JARVIS" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1">
   <icons>
-    <Image url="@/images/address_book_32.png" />
+    <Image url="@/images/png-transparent-jarvis-logo.png" />
```

Comment:
I'm not sure if we are able to use this image......

There might be some issues with reuse/copyright?

Reply:
I updated the image with a quick design so there shouldn't be any issues with the new logo now